clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/deploy/util/id/DcosSpinnakerAppId.groovy

```diff
@@ -32,9 +36,9 @@ class DcosSpinnakerAppId {
      *         Deemed unsafe because various Spinnaker components have trouble with a region with backslashes.
      *         <p/>
      *         Ex: {@code foo/bar}
-     * @see #getSafeRegion()
+     * @see #getSafeGroup()
      */
-    public String getUnsafeRegion() {
+    public String getUnsafeCombinedGroup() {
```

Comment:
Should these still not be region instead of group (i.e. getUnsafeRegion/getSafeRegion)? This is what we return for the "region" field on the models. I think the method you introduced called "getRegion" should be "getDcosCluster" instead. The documentation still refers to these as regions as well.

I guess it does make sense in that it represents the "group" of the marathon app itself, however (though this could change if we remove the cluster from the region).